tests/common/mellanox_data.py

```diff
@@ -242,6 +469,106 @@
         },
         "ports": {
             "number": 32
+        },
+        "thermals": {
+            "cpu_core": {
+                "start": 0,
+                "number": 2
+            },
+            "module": {
+                "start": 1,
+                "number": 32
+            },
+            "psu": {
+                "start": 1,
+                "number": 2
+            },
+            "cpu_pack": {
+                "number": 1
+            },
+            "asic_ambient": {
+                "number": 1
+            },
+            "port_ambient": {
+                "number": 1
+            },
+            "fan_ambient": {
+                "number": 1
+            },
+            "comex_ambient": {
+                "number": 1
+            }
         }
     }
 }
+
+THERMAL_NAMING_RULE = {
```

Comment:
These RULE dicts look like just text templates. Can we consider moving this definition to the test script file?
This mellanox_data.py file is for providing hardware related information that we cannot get from anywhere.